manifests/5.1/cluster-logging.v5.1.0.clusterserviceversion.yaml

```diff
@@ -327,6 +327,8 @@ spec:
                     value: "quay.io/openshift/origin-logging-fluentd:latest"
                   - name: CURATOR_IMAGE
                     value: "quay.io/openshift/origin-logging-curator5:latest"
+                  - name: IMAGE_LOG_METRIC_EXPORTER
+                    value: quay.io/openshift/origin-cluster-logging-operator:latest
```

Comment:
changes are going into 5.2 so this will need to be added to that file instead.. but i guess it will depend if https://github.com/openshift/cluster-logging-operator/pull/1007 lands first.  you might not need to do anything